src/Shared/AppLoadingView.xaml.cs

```diff
@@ -259,7 +259,16 @@ async void OutOfBox_CoreInstanceRegistered(object? sender, CoreInstanceEventArgs
 
                 core.CoreStateChanged += Core_CoreStateChanged;
 
-                await core.InitAsync();
+                try
```

Comment:
You'll want to do these same changes to the other non-oobe (lifetime) event handler just below this.